packages/atomic/cypress/e2e/facets/category-facet/category-facet-selectors.ts

```diff
@@ -40,9 +40,12 @@ export const CategoryFacetSelectors = {
   values() {
     return this.shadow().find('[part="values"]');
   },
-  childValue() {
+  linkValue() {
     return this.shadow().find('[part~="value-link"]');
   },
+  childValue() {
+    return this.values().find('[part~="value-link"]');
```

Comment:
Does it seem a problem that the differentiator (css selector) between both type of value is * that * subtle (for implementers) ?

Reply:
Good point, I would not bring changes here because, IMO, it may be interesting for v3 to rationalize the parts and their names with 2 things in mind:
 - We can now have the hierarchy described in the PR descriptions
 - We need to support all existing CSS customizations
 
Notably, I do not like the fact that I had to leave a "split" in the rendering logic depending on whether the response is hierarchical or not (as all response are hierarchical for the API I reckon, the flat ones are just 'hierarchical without children') . I would have preferred the rendering to always be "hierarchical", and that the lack of "hierarchy" comes from the lack of children, and nothing else (i.e. render children if any)

This was mostly caused by the existing parts and how they work.

I would suggest for v3:
 1.  "raze' existing parts
 2. ask ourselves "ok what makes sense to exposes"
 3. Compare with v2 & try to fill the obvious gaps
 4. Wait for complaints.